Jellyfin.Data/Entities/Libraries/Chapter.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Jellyfin.Data.Interfaces;
+
+namespace Jellyfin.Data.Entities.Libraries
+{
+    /// <summary>
+    /// An entity representing a chapter.
+    /// </summary>
+    public class Chapter : IHasConcurrencyToken
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Chapter"/> class.
+        /// </summary>
+        /// <param name="language">ISO-639-3 3-character language codes.</param>
+        /// <param name="startTime">The start time for this chapter.</param>
+        /// <param name="release">The release.</param>
+        public Chapter(string language, long startTime, Release release)
```

Comment:
This maybe needs a constructor with endTime as well? I'don't know how we handle optional arguments in other places.

Reply:
It actually makes sense, because the end time would be the start of the next chapter as far as I know.